pom.xml

```diff
@@ -68,6 +68,12 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+
+<!--        Placeholder for local import of an unpublished library to maven repo. (i.e this is a local repository)-->
```

Comment:
Nit, is the formatting correct here?

Reply:
Will remove this once we have beta version of ingest sdk with streaming API released.